GVRf/Framework/framework/src/main/java/org/gearvrf/GVRActivityBase.java

```diff
@@ -123,6 +121,7 @@ public void run() {
     abstract GVRActivityNative makeActivityNative();
     abstract GVRViewManager makeViewManager(GVRXMLParser xmlParser);
     abstract GVRMonoscopicViewManager makeMonoscopicViewManager(GVRXMLParser xmlParser);
+    abstract GoogleVRViewManager makeGoogleVRViewManager(GVRXMLParser xmlParser);
```

Comment:
It already is limited to that backend. We will generalize this fallback mechanism in the future. We also have variation in the monoscopic backends so it will be a gradual and possibly slow process.

Not registering is fine but just don't make it google vr specific; if you have to add a hidden setting in GVRConfigurationManager or something, for now. Let's just not add more dependencies from the framework to the backends. This drawFrameListener was a hack and there is something better in the works but again, I don't know when it will get to this branch.

Thanks for letting me know. Will look into this double-registration.